lineage/figures/figureS15.py

```diff
@@ -73,22 +76,22 @@ def makeFigure():
 
     ax, f = getSetup((10, 5), (2, 4))
 
-    ax[0].stackplot(times, control_L[0], control_L[1], control_L[2], labels=['state 1', 'state 2', 'state 3'], alpha=0.6)
-    ax[1].stackplot(times, conc1_L[0], conc1_L[1], conc1_L[2], labels=['state 1', 'state 2', 'state 3'], alpha=0.6)
-    ax[2].stackplot(times, conc2_L[0], conc2_L[1], conc2_L[2], labels=['state 1', 'state 2', 'state 3'], alpha=0.6)
-    ax[3].stackplot(times, conc3_L[0], conc3_L[1], conc3_L[2], labels=['state 1', 'state 2', 'state 3'], alpha=0.6)
+    ax[0].stackplot(times, control_L[:, 0], control_L[:, 1], control_L[:, 2], labels=['state 1', 'state 2', 'state3'], alpha=0.6)
```

Comment:
I fixed this in the last PR.

Reply:
Thank you.
Yes, I agree. I expected to see more changes, too.
I am working on figures in ODE and will update you as soon as it is ready.